Makefile.inc.in

```diff
@@ -9,7 +9,7 @@ endif
 
 AWK ?= @AWK@
 CC = @CC@
-CFLAGS = $(WARNFLAGS) $(NOWARNFLAGS) @CFLAGS@ -D LIBPREFIX=$(LIBPREFIX)
+CFLAGS = $(WARNFLAGS) $(NOWARNFLAGS) @CFLAGS@ -DLIBPREFIX=$(or $(LIBPREFIX),MDSplus)
```

Comment:
Hi Timo, the $(or "something", "else") command substitute "something" if it is non empty or "else" otherwise
should be working.. if we forgot to set LIBPREFIX it simply add MDSplus as a prefix
In any case we can remove it if you prefer

Reply:
akso i assime tge default MDSplus is only valid for one packjhage. would it not be better to add LIBPREFIX there. which os is this for.

Reply:
also, was the droppoied space intentional

Reply:
it was.. but just because I always found it without spaces .. could be a compatibility thing, I will google it :-)

Reply:
Timo, I thought the intent of this was to add a name for the internal version function.. however it is not save to leave the flag like -DLIBPREFIX=<empty>  so I think that MDSplus could be a solution as a generic prefix for versioning. 

Have you in mind a better solution for that? 

answering your question... the os I was compiling for is arch-linux with gcc 10.1